uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -270,8 +261,11 @@
     <string name="kh_uisdk_network_error">We seem to be having issues connecting to the network</string>
     <string name="kh_uisdk_new_ride">NEW RIDE</string>
     <string name="kh_uisdk_next">Next</string>
+    <string name="kh_uisdk_no_availability">We’re not live here yet! Get in touch if you have suggestions for local fleets.</string>
+    <string name="kh_uisdk_no_availability_subtitle">Please select another time</string>
+    <string name="kh_uisdk_no_availability_title">No availability for the requested time</string>
     <string name="kh_uisdk_no_destination">No Destination</string>
-    <string name="kh_uisdk_no_results_found">Apologies ASAP bookings are not currently available for this destination, however you can book in advance for your next trip!</string>
```

Comment:
These values should be the other way around. The long text should be the one to stay. Valid for all languages

Reply:
The long text was moved above in the K3002 string